### DISCUSSION TITLE
WebRTC example in node.js doesn't work

### DISCUSSION THREAD

**Discussion by askarzh** [libp2p/js-libp2p#2535] [Q&A]
Hello!

  I'm trying to start example from README for WebRTC transport in node.js and I can't do as it is always complain on webRTC module:
Uncaught TypeError TypeError: transport.filter is not a function

Regards,

**Comment by achingbrain** [comment]
Which README are you following?  Can you post some code for how you are configuring your node?

**Comment by achingbrain** [comment]
Closing due to lack of response.
